Makefile

```diff
@@ -90,6 +90,9 @@ kind-cluster: ## Standup a kind cluster for e2e testing usage
 	${KIND} delete cluster --name ${KIND_CLUSTER_NAME}
 	${KIND} create cluster --name ${KIND_CLUSTER_NAME}
 
+image-registry: ## setup in-cluster image registry 
```

Comment:
super minor nit: it looks like all of the other comments that drive the `make help` output start with a capitalized letter.

Reply:
I change it.  Thanks!